autoprotocol/instruction.py

```diff
@@ -554,14 +555,7 @@ class Incubate(Instruction):
     """
 
     WHERE = [
-        "ambient",
-        "warm_30",
-        "warm_35",
-        "warm_37",
-        "cold_4",
-        "cold_20",
-        "cold_80",
-        "cold_196",
+        allowable.strip("_") for allowable in Location.__dict__.get("_member_names_")
```

Comment:
I noticed "warm_30" is included in this list comprehension but I don't see it in the docstring...not sure if that matters. 
<img width="1781" alt="image" src="https://user-images.githubusercontent.com/60940557/211679168-4c5984e6-99cd-4362-a933-2375aab7dcef.png">

Reply:
I just replaced all the strings that I saw in the list. I'll add it to the docstring